### PR TITLE
Remove billingAddress and shippingAddress

### PR DESCRIPTION
**Pull request by Konafets**
The Customer entity does not have have the mentioned properties.

| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/book/customers/customer_and_shopuser.rst [hunk 1]
```diff
@@ -28,8 +28,7 @@ As usual, use a factory. The only required field for the Customer entity is ``em
 
    $this->container->get('sylius.repository.customer')->add($customer);
 
-The Customer entity can of course hold other information besides an email, it can be for instance
-``billingAddress`` and ``shippingAddress``, ``firstName``, ``lastName`` or ``birthday``.
+The Customer entity can of course hold other information besides an email, it can be for instance ``firstName``, ``lastName`` or ``birthday``.
```
Comment (Zales0123) [inline docs/book/customers/customer_and_shopuser.rst]: Customer has `defaultAddress` and `addresses` collection, maybe we should mention them here, just to keep some information about customer addresses? :)

Comment (Konafets) [reply docs/book/customers/customer_and_shopuser.rst]: This brings up an overall problem of the documentation. The information is spread over all the components and bundles. In my opinion the information that there are other fields has no real value to the reader at this point. If you mention the `addresses` collection you should also mention the AddressBook and that the addresses only shown to the customer in his / her profile but not in the admin panel.

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "patch-5"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by pamil** [comment]
Thanks, Stefano! :1st_place_medal:

**Pull request merged**